run-py-tests.sh

```diff
@@ -4,6 +4,7 @@ python3 -m venv venv
 # shellcheck disable=SC1091
 . ./activate
 pip3 install .
+pip3 install ".[dev]"
```

Comment:
FYI, you can actually remove `pip install .` if you're doing `pip install ".[dev]"`.

Reply:
oh didn't know that, thanks